### ISSUE TITLE
Change Spinner Loader to progress bar when document is loading

### ISSUE THREAD

**Issue by MilesYM** [react-pdf-viewer/react-pdf-viewer#232]
I've been using your pdf viewer for a couple of months now and it's working like a charm. Thank you for the great work.

Some of the files displayed on our platform can be pretty heavy ( > 20Mo) and they can take up to 20-30s to load depending on the connection.

We would like to implement a progress bar to show how much of the document has been downloaded. Then when it gets to 100%, the pdf viewer starts.
Is it possible? If yes how?
Right now there is only a spinner. But if the document is heavy, it can take a while and some of our users think there is a bug and leave the page before the document has been fully loaded.

Thank you very much.

**Comment by phuoc-ng** [comment]
It's good suggestion, @MilesYM.
I'll add support for that in the next version.

**Comment by MilesYM** [comment]
Thank you so much for your reactivity.

Can't wait!

**Comment by MilesYM** [comment]
Hi there, I wanted to follow up with you on this feature. Is there any chance we could see this feature coming soon? Thank you.

**Comment by phuoc-ng** [comment]
@MilesYM 
It was implemented and will be available in the next big release 2.0.

_Demo_

_With custom loader_: You'll see a custom blue progress bar

![custom-loader](https://user-images.githubusercontent.com/57786711/96359511-ece46380-113d-11eb-8191-799c098f2ef8.gif)

_With the default loader (`Spinner`)_

![default-loader](https://user-images.githubusercontent.com/57786711/96359509-dfc77480-113d-11eb-86a3-80c224f07154.gif)

**API**

``` javascript
import { ProgressBar, Viewer } from '@react-pdf-viewer/core';

<Viewer
    renderLoader={(percentages: number) => (
        // You can use your own progress bar component
        <div style={{ width: '240px' }}>
            <ProgressBar progress={Math.round(percentages)} />
        </div>
    )}
/>
```

v2.0.0 is really coming soon, you'll receive an update email in next a few days.

**Comment by phuoc-ng** [comment]
@MilesYM 
v2.0.1 was released. Please take a look at
* The upgrade guide https://react-pdf-viewer.dev/updates
* Customize loader renderer example: https://react-pdf-viewer.dev/examples/customize-loader-renderer

Let me know if you have any questions/issues.
